be/src/formats/parquet/stored_column_reader.cpp

```diff
@@ -477,14 +549,43 @@ Status RequiredStoredColumnReader::read_records(size_t* num_records, ColumnConte
     return Status::OK();
 }
 
-Status RequiredStoredColumnReader::_next_page() {
+Status RequiredStoredColumnReader::_next_selected_page(size_t records_to_read, size_t* records_to_skip) {
```

Comment:
Is there any difference to `OptionalStoredColumnReader::_next_selected_page`?

Reply:
> Is there any difference to `OptionalStoredColumnReader::_next_selected_page`?

No difference, I have move the function to the base class.